lib/base/ignore.h

```diff
@@ -0,0 +1,13 @@
+#ifndef LIB_BASE_IGNORE_H_
+#define LIB_BASE_IGNORE_H_
+
+namespace base {
+
+// Idiom to ignore an otherwise unused variable or result.
+// Prefer to (void) cast when dealing with [[nodiscard]].
```

Comment:
When you say "Prefer to (void) cast" is it just for readability, or is there some other advantage?

Reply:
Just readability and consistency. C style casts are otherwise unused in the Google C++ style.

Reply:
Where can I read about Google C++ style?

Reply:
Should've known that Goole will know... https://google.github.io/styleguide/cppguide.html